LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
revert --- we can discuss keeping the license in sync with the rest of semantic later